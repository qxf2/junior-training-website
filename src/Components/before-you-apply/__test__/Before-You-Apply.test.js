import React from 'react';
import ReactDOM from 'react-dom';
import Before from '../Before-You-Apply';

import { render, cleanup } from '@testing-library/react';
import "@testing-library/jest-dom/extend-expect";

import renderer from "react-test-renderer";

afterEach(cleanup);

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Before/>, div);
});

it("renders before you apply file correctly", () => {
   const { getByTestId }  = render(<Before/>)
   expect(getByTestId('before-you-apply'));
});

it("before-you-apply.js file matches snapshot", () =>{
  const tree = renderer.create(<Before/>).toJSON();
  expect(tree).toMatchSnapshot();
});