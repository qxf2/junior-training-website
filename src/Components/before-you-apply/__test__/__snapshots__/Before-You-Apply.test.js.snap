// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`before-you-apply.js file matches snapshot 1`] = `
<section
  data-testid="before-you-apply"
  id="before-you-apply"
>
  <div
    className="react-reveal row"
    style={
      Object {
        "opacity": undefined,
      }
    }
  >
    <h4
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      Before you apply!
    </h4>
    <div
      className="twelve columns collapsed"
    >
      <div
        className="bgrid-quarters s-bgrid-thirds cf"
      >
        <ul>
          <li>
             Fully remote 
          </li>
          <li>
             Not for everyone 
          </li>
          <li>
             Fast pace - we will not wait for you 
          </li>
          <li>
             No placement guarantee 
          </li>
          <li>
             No money back 
          </li>
        </ul>
      </div>
    </div>
  </div>
</section>
`;
